tests/data/return_annotation_brackets.py

```diff
@@ -0,0 +1,134 @@
+# Control
+def double(a: int) -> int:
+    return 2*a
+
+# Remove the brackets
+def double(a: int) -> (int):
```

Comment:
Also add an example with multiple sets of parens. And maybe for additional fun, what if there is more whitespace between the parentheses?